!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  block_size: 13
  disp12MaxDiff: -1
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      block_size: 13
      disp12MaxDiff: -1
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      min_disparity: 4
      mode: false
      name: Default
      num_disparities: 128
      p1: 19
      p2: 19
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      prefilter_cap: 10
      prefilter_size: 11
      sgbm: false
      smaller_block_size: 0
      speckle_range: 4
      speckle_window_size: 500
      state: true
      texture_threshold: 0
      type: ''
      uniqueness_ratio: 5.0
      x_skew: -4.5
      y_offset: 26
      y_skew: 0.0
    state: []
  min_disparity: 4
  mode: false
  num_disparities: 128
  p1: 19
  p2: 19
  prefilter_cap: 10
  prefilter_size: 11
  sgbm: false
  smaller_block_size: 0
  speckle_range: 4
  speckle_window_size: 500
  texture_threshold: 0
  uniqueness_ratio: 5.0
  x_skew: -4.5
  y_offset: 26
  y_skew: 0.0
state: []
